doc/man5/config.pod

```diff
@@ -265,8 +265,10 @@ Specifies the pathname of the module (typically a shared library) to load.
 
 =item B<activate>
 
-If present, the module is activated. The value assigned to this name is not
-significant.
+If present and set to one of the values yes, on, true or 1, then the associated
+provider will be activated.  Omitting this item, or setting it to
+any other value, will prevent the associated provider from being activated.
+Note that the above activation values are case sensitive
```

Comment:
I wonder if they should be case insensitive or not?

Reply:
Please let's not overengineer it. Lower case is perfect imho. Do you agree?

 -[ ] YeS
 -[ ] nO

Reply:
The documentation change explicitly indicates settings are case sensitive